flow/display_list.h

```diff
@@ -0,0 +1,427 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLOW_DISPLAY_LIST_H_
+#define FLUTTER_FLOW_DISPLAY_LIST_H_
+
+#include "third_party/skia/include/core/SkBlurTypes.h"
+#include "third_party/skia/include/core/SkCanvas.h"
+#include "third_party/skia/include/core/SkColorFilter.h"
+#include "third_party/skia/include/core/SkImage.h"
+#include "third_party/skia/include/core/SkImageFilter.h"
+#include "third_party/skia/include/core/SkPicture.h"
+#include "third_party/skia/include/core/SkShader.h"
+#include "third_party/skia/include/core/SkVertices.h"
+
+// The Flutter DisplayList mechanism is used in place of the Skia SkPicture
+// mechanism. It encapsulates a sequence of rendering operations into a
+// persistent list that can be replayed upon request by calling its
+// dispatch() method with a Dispatcher object that responds to the various
+// rendering methods encapsulated therein. The mechanism is inspired by
+// the SkLiteDL class that is not directly supported by Skia, but has been
+// recommended as a basis for custom display lists for a number of their
+// customers.
```

Comment:
I'd probably omit reference to SkPicture here, or maybe put it further down.  This paragraph is also a bit verbose. Maybe something just like:

```
The Flutter DisplayList encapsulates a persistent sequence of rendering operations.
```

And then maybe a paragraph at the end with references to implementations of `Dispatcher` and Skia's codebase.

Reply:
I reworked the comment.

Reply:
Is the new comment better?

Reply:
Yes, thanks